doc/release/1.15.0-notes.rst

```diff
@@ -164,25 +187,33 @@ both numpy 1.15 and 1.16. For more information on how to update code to account
 for this future change see "basics/structured arrays/accessing multiple fields"
```

Comment:
Can this be made into a direct link?

Reply:
Not sure where to send the link. There is a section in the 1.14 documentation, incorrectly claiming the change will go into 1.15, but I'm not sure how to access that section. See https://docs.scipy.org/doc/numpy/user/basics.rec.html, but is it safe to reference the docs? OTOH, there is the file `numpy/doc/structured_arrays.py`.

Reply:
Yes, I meant a link to the docs; I don't think it hurts - if it breaks it is still no worse than it is now. Also, I think `Accessing Multiple Fields` might just work... (I think sphinx allows linking to section titles, but may be wrong...)

Reply:
I think sphinx requires specific section labels.